.github/workflows/universal-cloud-detect-changes.yml

```diff
@@ -0,0 +1,56 @@
+name: Detect Changes
```

Comment:
@matejsemancik Tady jen přemýšlím jestli by se to nemělo jmenovat `kmp-cloud-detect-changes`. Není to totiž použitelné na iOS only repo jako například ten `universal-cloud-backup` kvůli té path - `androidApp` a `iosApp`.

Reply:
Makes sense, díky, upravím